CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [1.2.4](https://github.com/fgrehm/vagrant-lxc/compare/v1.2.3...v1.2.4) (Dec 12, 2017)
+
+  - Support alternative `lxcpath` [[GH-413]]
```

Comment:
Sorry, do u mind creating the links as well for the issues? I just noticed that some of the ones in the other changelog entries are broken :( I fixed some of them [in master now](https://github.com/fgrehm/vagrant-lxc/commit/b96ba86f72e277accd826268583d45ee10299ad5)